### DISCUSSION TITLE
Provide AST viewer for SWC

### DISCUSSION THREAD

**Discussion by magic-akari** [swc-project/swc#9646] [Ideas]
It's enough to provide a wasm target.

This is a simple implementation. We can move them to the swc/bindings, so we can use it in the swc playground.
https://github.com/magic-akari/swc-ast-viewer/blob/main/crates/swc_ast_viewer/src/lib.rs

**Comment by CPunisher** [comment]
It is also implemented in https://ast.sxzz.moe/

**Reply by magic-akari** [reply]
SWC's JSON serialization output differs from the original structure. 
Therefore, I prefer to use SWC's debug output, which is closer to the structure of structs/enums in SWC.
